packages/jest-cli/src/__tests__/TestRunner-test.js

```diff
@@ -38,6 +38,100 @@ test('.addReporter() .removeReporter()', () => {
   expect(runner._dispatcher._reporters).not.toContain(reporter);
 });
 
+// Checking for the method _addCustomReporters
+// Custom reporters used here are the reporters within the package
+// No extra reporters are included to be used
+describe('_addCustomReporters', () => {
```

Comment:
@thymikee Hey! I am new to testing. Wrote some unit tests to check the integration. Would you mind giving it a review?